services/storage/src/simcore_service_storage/api/rpc/_simcore_s3.py

```diff
@@ -38,8 +44,14 @@ async def start_export_data(
     task_manager: TaskManager,
     job_filter: AsyncJobFilter,
     paths_to_export: list[PathToExport],
+    export_as: Literal["path", "download_link"],
 ) -> AsyncJobGet:
-    task_name = export_data.__name__
+    if export_as == "path":
+        task_name = export_data.__name__
+    elif export_as == "download_link":
+        task_name = export_data_as_download_link.__name__
+    else:
+        raise ValueError(f"Invalid export_as value: {export_as}")
```

Comment:
Q: why is the export_as not passed as an argument to the task that is submitted?